src/main/java/com/jorgepatrick/RomanNumbers.java

```diff
@@ -0,0 +1,10 @@
+package com.jorgepatrick;
+
+public class RomanNumbers {
+    public static final String I = "I";
+    public static final String II = "II";
+    public static final String III = "III";
+    public static final String IV = "IV";
+    public static final String V = "V";
+    public static final String VI = "VI";
```

Comment:
Maybe it should have only the symbols. Like `I` and `V`. Who is using them can compose the number.

Reply:
But wont that make it harder to read i.e.: Arguments.of(I + I, I, I + I + I)?